.github/workflows/CI.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         logstash: [ "7.16.3", "7.17.1", "8.3.2" ]
-        opensearch: [ "1.2.1" ]
+        opensearch: [ "1.3.4", "2.1.0" ]
```

Comment:
Does this mean we don't want the `1.2.1` version at all?

Reply:
Other clients tests against all released version.

Reply:
yes, we want to run the tests only on the latest released versions of 1.x and 2.x